### PR TITLE
[docs] Reorganize information on SSL certificates

### PR DESCRIPTION
**Pull request by jugglinmike**
The project's top-level `README.md` file contains a good deal of
information which may not be relevant to all readers. This risks
overwhelming newcomers, and it also discourages referencing the content
in more targeted contexts from the project's documentation website.

Move the information on SSL certificates from the top-level `README.md`
file to the corresponding document within the `tools/certs/` directory.
Reference the newly-expanded file from the `docs/` directory so that it
is included in the generated website (including the search engine).

This is in service of gh-5289.

### CODE REVIEW

**Review by Ms2ger — APPROVED** [review]
Makes sense to me, but should we consider adding a pointer from the README to this new document?

**Comment by jugglinmike** [comment]
Thanks, @Ms2ger. Linking to the new document could help some people find that content, but I don't know how to make that decision generally. At the extreme, we would like to every document from the readme. That would be tough to maintain, though anything less would require some more subjective criteria about the relative importance of the topics we cover.

File: tools/certs/README.md [hunk 1]
```diff
@@ -1,11 +1,76 @@
...
+For browsers that use the Certificate Authorities of the underlying OS, such as
+Chrome and Safari, you need to adjust the OS. For macOS, go to Keychain Access
+and add the certificate under **login**.
+
+**NOTE**: The CA should not be installed in any browser profile used
+outside of tests, since it may be used to generate fake
+certificates. For browsers that use the OS certificate store, tests
+should therefore not be run manually outside a dedicated OS instance
+(e.g. a VM). To avoid this problem when running tests in Chrome or
+Firefox use `wpt run`, which disables certificate checks and therefore
```
Comment (sideshowbarker) [inline tools/certs/README.md]:
```suggestion
Firefox, use `wpt run`, which disables certificate checks and therefore
```

File: tools/certs/README.md [hunk 2]
```diff
@@ -1,11 +1,76 @@
...
+doesn't require the root CA to be trusted.
+
+## Regenerating certificates
+
+If you wish to generate new certificates for any reason it's possible to use
```
Comment (sideshowbarker) [inline tools/certs/README.md]:
```suggestion
If you wish to generate new certificates for any reason, it's possible to use
```

File: tools/certs/README.md [hunk 3]
```diff
@@ -1,11 +1,76 @@
...
+
+After installation, ensure that the path to OpenSSL (typically,
+this will be `C:\OpenSSL-Win32\bin`) is in your `%Path%`
+[Environment Variable](http://www.computerhope.com/issues/ch000549.htm).
+If you forget to do this part, you will most likely see a 'File Not Found'
+error when you start wptserve.
 
-### Updating these certs
+Finally, set the path value in the server configuration file to the
+default OpenSSL configuration file location. To do this create a file
```
Comment (sideshowbarker) [inline tools/certs/README.md]:
```suggestion
default OpenSSL configuration file location. To do this, create a file
```

**Review by sideshowbarker — APPROVED** [review]
Sprinkle in a few commas

**Comment by jugglinmike** [comment]
Thanks @Ms2ger and @sideshowbarker!

**Pull request merged**